scripts/vectorizer-load-test/README.md

```diff
@@ -0,0 +1,46 @@
+# vectorizer load test
+
+This repo contains scripts for load testing vectorizer with a large dataset.
+
+
+We will use this dataset: https://huggingface.co/datasets/Cohere/wikipedia-22-12
+
+The dataset is already chunked. We will load the dataset into one table and then
+"de-chunk" it into a second table. This second table can be used for vectorizer
+load tests. There are 8.59M rows in the original dataset.
+
+## 1. Preparing the environment
+In strict order, follow:
+
+1. Add a `.env` file and put a `DB_URL` in it. The value should be a Postgres DB connection URL. It can be a local DB or a remote DB.
+2. Run `./0_init.sh`. This will install and set up all dependencies required for running this project.
+
+## 2. Generating a Vectorizer and its dataset
+In order to generate a vectorizer, we first need to prepare its dataset from where the embeddings will be created.
+
+In strict order, follow:
+
+1. Run `source venv/bin/activate`. This will activate the Python virtual env in which dependencies have been installed.
+2. Run `python3 1_load.py`. This will dump all the dataset into a new table called `wiki_orig`. As the dataset is pretty large, feel free to cancel the script at any moment in case you want a smaller dataset.
+3. Run `./2_execute_sql.sh sql/1_prepare.sql`. This will create a new table called `wiki`, populated from `wiki_orig`. It is a dechunked version of `wiki_orig`, and will become the source table for the embeddings work.
+4. (optional) run `./utils/dump_db.sh`. This will create a dump of your DB. In case you wanna run several tests, you can use it later in order to restore the initial status. Use `./utils/restore_db.sh` to restore it at any point.
+5. run `./2_execute_sql.sh sql/2_test.sql`. This will trigger (not executing) the whole embeddings pipeline. Internally it does several stuff:
```

Comment:
```suggestion
5. run `./2_execute_sql.sh sql/2_test.sql`. This will trigger (not executing) the whole embeddings pipeline. Internally it does several things:
```